README.md

```diff
@@ -1,2 +1,11 @@
 # screencrash-components
 Repository for the loose components (screen, audio, robot etc.) of Screencrash (also known as Skärmkrock)
+
+## Running components
+You can use make to run individual (or all) components. See table below.
+
+| Command         | Result                                                                                    |
+| --------------- | ----------------------------------------------------------------------------------------- |
+| make -j2 dev    | Run all components (audio, screen) in parallell. Log output will be to the same terminal. |
```

Comment:
I considered this for the main repo, but went with another solution to make it a bit easier to use. Did you check it, and what do you think?

Reply:
Hmm, I checked now. The main problem I see is that Windows don't have `sh` but if we can find a good alternative solution for that then it would be nice.
Sidenote: Where do printouts end up when you run the program in the background? Perhaps not super-important when running live but could be nice to see when developing :)

Reply:
Printouts end up in the same terminal, even if you continue to run other commands. So it can be annoying if you're not careful :)

I assumed whatever way you got 'make' would also provide sh, but apparently not. Does it provide any shell?

Reply:
I'm using [this one](http://gnuwin32.sourceforge.net/packages/make.htm) and unfortunately it only provides a make binary (no shell).

Reply:
It would be nice to have something similar to tmux which spawns up all processes in different tabs/windows. But then again we have Windows not supporting tmux :(

Reply:
Maybe we should do it with -j then, both here and in the main repo. Seems like it would be easy to spend a lot of time on this otherwise.